README.md

```diff
@@ -1,4 +1,3 @@
 StopWatch
 =========
-
-A timer class in C++
```

Comment:
OK. So code review comments do show up in the git channel. That's probably too chatty so I'll turn it off.